sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/FullBackupOperation.java

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.keyvault.administration.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The FullBackupOperation model. */
+@Fluent
+public final class FullBackupOperation {
```

Comment:
You should have a single `BackupOperation` for both full- and selective-backup. The URI varies, but the functionality is exactly the same.

Reply:
Actually we only have one `BackupOperation`. This is auto-generated code though, so to change the name of the operation we would need to add a custom directive to the AutoRest config file and that is discouraged. Please refer to [this comment](https://github.com/Azure/azure-sdk-for-java/pull/12405#discussion_r448671219) for more details.

Reply:
Turns out we can use directives to rename models :)